src/project/project.controller.ts

```diff
@@ -71,4 +76,32 @@ export class ProjectController {
     response.setHeader('Content-type', 'text/csv');
     response.send(csv);
   }
+
+  @Post('/projects/feedback')
+  async receiveFeedback(@Req() request: Request, @Res() response) {
+    const recaptcha = new RecaptchaV2(this.config.get('RECAPTCHA_SITE_KEY'), this.config.get('RECAPTCHA_SECRET_KEY'));
+
+    recaptcha.verify(request, async (error, data) => {
+      if (!error) {
+        const { projectid, projectname, text } = request.body;
+        try {
+          // receiveFeedback uses octokit to create issues on our dcp-zap-data-feedback repository
+          await this.projectService.receiveFeedback(projectid, projectname, text);
```

Comment:
I wonder if the name of this method could be a little clearer? It really actually sends the feedback to Github, right?

Reply:
Yes! Agreed, will fix

Reply:
Fixed!